### DISCUSSION TITLE
Commits no longer trigger actions

### DISCUSSION THREAD

**Discussion by codehag** [community/community#25021] [Actions]
<p>Hi,</p>
<p>I have a github action set up on <a href="https://github.com/mozilla-spidermonkey/jsparagus" rel="nofollow noopener">https://github.com/mozilla-spidermonkey/jsparagus</a></p>
<p>However, it is no longer running the actions as of five days ago. It works fine for me on my fork. Any idea what is going on?</p>

**Comment by Maxence-Perrin** [comment]
<p>Hi,</p>
<p>I have the same issue since 3pm on my private repository.</p>
<p>Everything worked well before 3pm.</p>

**Comment by vlasscontreras** [comment]
<p>Just noticed this too, also in a private repo (where I just added 2 actions).</p>
<p>After PR (build) and pushing tags (release) neither of both ran and in the actions tab I see this despite I have created <strong>.github/workflows/build.yml</strong> and <strong>.github/workflows/release.yml</strong> just like in other repos:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e961ba4e7d9b37511e0e1168c2901c36905aea51.png" data-download-href="/uploads/short-url/xiAD90nhmrZpfNDJRMEQNIkviVP.png?dl=1" title="github.com_X_R_actions.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/17748144/181088461-4c7ca3b7-5d4e-418e-8e5e-2bb88b68a084.png" alt="github.com_X_R_actions.png" data-base62-sha1="xiAD90nhmrZpfNDJRMEQNIkviVP" width="690" height="293" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e961ba4e7d9b37511e0e1168c2901c36905aea51_2_690x293.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e961ba4e7d9b37511e0e1168c2901c36905aea51_2_1035x439.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/e961ba4e7d9b37511e0e1168c2901c36905aea51_2_1380x586.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e961ba4e7d9b37511e0e1168c2901c36905aea51_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.com_X_R_actions.png</span><span class="informations">4315×1837 160 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment]
<p>I noticed that your workflow was triggered 7 hours ago, the time is after you reported this ticket.</p>
<p>Looks like it can be triggered mormally. Can you check if it has back to work now?</p>
<p>And also please confirm if you are the collaborator of this repository.</p>

**Comment by codehag** [comment]
<p>It triggered a few times yesterday but it is not consistent. It stopped working again shortly after. We need it to run with every push. At the moment we don’t have more information about why it does or doesn’t trigger.</p>
<p>I see two things that might be happening</p>
<ol>
<li>merged from a fork might not work</li>
<li>I don’t know if perhaps it is the frequency of pushing. Actions say that we can have 20 concurrent actions, and so far we have not had that happen to my knowledge. Perhaps there is another limit?</li>
</ol>

**Comment by codehag** [comment]
<p>So, for my case, I seem to have confirmed that it only runs the tests if we are pushing to master. If we push to a fork, it doesn’t show the results in the upstream, only in the fork. Is there a way to change this?</p>

**Comment by Maxence-Perrin** [comment]
<p>I noticed my trigger pull request is no longer triggered.</p>
<p>Push is trigerred normally.</p>
<p>Maybe it can help.</p>

**Comment by codehag** [comment, marked as answer]
<p>Do you have a fork that you are making the pull request from?</p>
<p>At the moment this is what we are trying: <a href="https://github.com/mozilla-spidermonkey/jsparagus/pull/202" rel="nofollow noopener">https://github.com/mozilla-spidermonkey/jsparagus/pull/202</a></p>

**Comment by Maxence-Perrin** [comment]
<p>I didnt fork, i just create a pull request from another branch.</p>
<p>My trigger looks like : on: [pull_request] (I remove all parameters to test).</p>
<p>If i change my trigger to on: [push], it will be trigger normally</p>

**Comment by codehag** [comment]
<p>our issue is resolved with what I did above. It looks like you might have something different… Without seeing the code its hard to say.</p>
